### PR TITLE
Dice

### PR DESCRIPTION
**Pull request by xSpacklesx**
I present for your reviewing pleasure, the dice rolling function

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -71,22 +71,26 @@ async def on_member_join(member):
...
-    # raise discord.DiscordException
-    # old/
-
-    response = f"Howdy {ctx.author}"
-    await ctx.send(response)
+@bot.command(name="roll")
+async def roll(ctx, die: str):
+    try:
+        num_of_dice, die_sides = die.split("d")
+        print(num_of_dice)
```
Comment (RayZhengGit) [inline bot.py:78]: are these print statements still needed?

File: bot.py [hunk 2]
```diff
@@ -71,22 +71,26 @@ async def on_member_join(member):
...
+        int(die_sides)
+    except ValueError:
+        await ctx.send("Error enter ints \nuse command  like !roll 2d20")
+        return
+
+    i = 0
+    while i < int(num_of_dice):
+        roll_result = str(random.randint(1, int(die_sides)))
+        i += 1
+        await ctx.send("Rolling: d" + str(die_sides) +
```
Comment (RayZhengGit) [inline bot.py:90]: could you put the "Rolling: d" + str(die_sides)" outside of the while loop so it only prints once and maybe just say that it is rolling three of that die to reduce chat spam.

Comment (RayZhengGit) [reply bot.py:90]: example:
Rolling 3d20
Rolled: 1
Rolled: 5
Rolled: 6

**Review by RayZhengGit — DISMISSED** [review]

File: bot.py (hunk 1 above)
Comment (xSpacklesx) [reply bot.py:78]: Yeah rn I'm using them to log to the console what the user tries to roll
I'm eventually gonna output that to a log file but for now, I'm print logging

**Review by dsmarcot2018 — APPROVED** [review]
I think it looks all good to me!

**Review by Joe-Marchesini — APPROVED** [review]
All looks good, except like what Ray said, we should try to reduce chat spam.

**Pull request merged**